app/org/sagebionetworks/bridge/dynamodb/DynamoUploadDao.java

```diff
@@ -201,9 +204,11 @@ public Upload getUpload(@Nonnull String uploadId) {
 
         String nextPageOffsetKey = (page.getLastEvaluatedKey() != null) ? page.getLastEvaluatedKey().get(UPLOAD_ID).getS() : null;
         
-        return new ForwardCursorPagedResourceList<>(uploadList, nextPageOffsetKey, pageSize)
-                .withFilter("startDate", startTime.toString())
-                .withFilter("endDate", endTime.toString());
+        return new ForwardCursorPagedResourceList<>(uploadList, nextPageOffsetKey)
+                .withRequestParam(ResourceList.OFFSET_KEY, offsetKey)
+                .withRequestParam(ResourceList.PAGE_SIZE, pageSize)
+                .withRequestParam(ResourceList.START_TIME, startTime)
+                .withRequestParam(ResourceList.END_TIME, endTime);
```

Comment:
The filters are new, so no, no need to use the old names here. Migrating requires changing the parameters used (both old and new are supported by the controllers).